### ISSUE TITLE
Add more information for the solutions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jsvazic/double-fanucci-ga#4]
```
When displaying the results, might want to list: 

a) Total number of cards 
b) the total value of all the calculated stacks added together.


```

Original issue reported on code.google.com by `jsva...@gmail.com` on 8 May 2009 at 6:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `jsva...@gmail.com` on 8 May 2009 at 6:44
- Added labels: **Milestone-Release1.1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jsva...@gmail.com` on 9 May 2009 at 3:35
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed for v1.1.  It should be interesting to see what the feedback of the 
labels will 
be.
```

Original comment by `jsva...@gmail.com` on 9 May 2009 at 5:14
- Changed state: **Fixed**
